README.md

```diff
@@ -1,11 +1,45 @@
-[![lerna](https://img.shields.io/badge/maintained%20with-lerna-cc00ff.svg)](https://lerna.js.org/)    [![build](https://img.shields.io/travis/wizeline/access-decision-manager/master.svg)](https://travis-ci.org/wizeline/access-decision-manager)
-
+[![lerna](https://img.shields.io/badge/maintained%20with-lerna-cc00ff.svg)](https://lerna.js.org/) [![build](https://img.shields.io/travis/wizeline/access-decision-manager/master.svg)](https://travis-ci.org/wizeline/access-decision-manager)
 
 ### Access-decision-manager.
 
-This repo maintains the following npm packages 
+This repo maintains the following npm packages
 
-* [access-decision-manager](https://github.com/wizeline/access-decision-manager/tree/master/packages/access-decision-manager)
-* [access-decision-manager-express](https://github.com/wizeline/access-decision-manager/tree/master/packages/access-decision-manager-express)
+- [access-decision-manager](https://github.com/wizeline/access-decision-manager/tree/master/packages/access-decision-manager)
+- [access-decision-manager-express](https://github.com/wizeline/access-decision-manager/tree/master/packages/access-decision-manager-express)
 
 Their main objective is to determine access rights based on the [Voter](https://symfony.com/doc/current/security/voters.html) pattern
+
+### Getting Started
+
+If you'd like to contribute to this project, a great place to get started it so checkout the project and make sure that the tests are working for you.
+
+To do this the following steps can be followed:
+
+1. Clone the project.
+   ```bash
+   git clone git@github.com:wizeline/access-decision-manager.git
+   ```
+2. Install the dependencies. We use [`npm ci`](https://docs.npmjs.com/cli/v8/commands/npm-ci) to ensure the same packages are installed on all machines.
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**question:** ‏
It wasn't clear to me where should I do this, first I did it at root level and then realized I had to do it also at the package level. Shouldn't this be handle by lerna? or should we just clarify that you need to cd to each package and run `npm ci` too there?

Reply:
You "shouldn't need" to run `npm ci` in the packages.  Because the root `package.json` file has `"install": "lerna bootstrap"`, after running `npm ci` NPM should call this post-install hook, which should output something like:
```
> root@undefined install /Users/mfarmer/Sites/wizeline/access-decision-manager
> lerna bootstrap

lerna notice cli v3.15.0
lerna info Bootstrapping 2 packages
lerna info Installing external dependencies
lerna info Symlinking packages and binaries
lerna success Bootstrapped 2 packages
added 1048 packages in 23.431s
```

With this we should now see child `node_modules` folders:
```
$ ls packages/access-decision-manager-express/node_modules
@wizeline         http-status-codes
```

---

If you run `for f in `find . -name 'node_modules'`; do rm -rf $f; done` it will remove all `child node_module` folders, and I think running these instructions should get the tests to all work just fine.

---

It's possible I'm missing something you are trying to do here, so maybe you could run these commands and then post the error you are seeing and I could better understand.

I think we should move to something like turborepo/nx, etc, and so hopefully these lerna difficulties will go away then.

Reply:
I managed to replicate it by removing the node_modules and running again `npm ci` and then `npm test`.
I do get the same lerna bootstrap message and dependencies installed:
```
> install
> lerna bootstrap

lerna notice cli v3.15.0
lerna info Bootstrapping 2 packages
lerna info Installing external dependencies
lerna info Symlinking packages and binaries
lerna success Bootstrapped 2 packages

added 1048 packages, and audited 1115 packages in 15s
```
but then this test fails:
```FAIL src/__tests__/access-decision-manager-provider.test.ts
  ● Test suite failed to run

    Cannot find module '@wizeline/access-decision-manager' from 'access-decision-manager-provider.test.ts'

      1 | // eslint-disable-next-line import/no-extraneous-dependencies
      2 | import { Request } from 'express';
    > 3 | import AccessDecisionManager from '@wizeline/access-decision-manager';
        | ^
```
then I do the following to fix it:
```
cd packages/access-decision-manager-express
npm ci
added 2 packages, and audited 3 packages in 1s

found 0 vulnerabilities
```

Reply:
I see, this fixes the error 

`
diff --git a/packages/access-decision-manager-express/src/__tests__/access-decision-manager-provider.test.ts b/packages/access-decision-manager-express/src/__tests__/access-decision-manager-provider.test.ts
index 0972bc9..9c93551 100644
--- a/packages/access-decision-manager-express/src/__tests__/access-decision-manager-provider.test.ts
+++ b/packages/access-decision-manager-express/src/__tests__/access-decision-manager-provider.test.ts
@@ -1,6 +1,6 @@
 // eslint-disable-next-line import/no-extraneous-dependencies
 import { Request } from 'express';
-import AccessDecisionManager from '@wizeline/access-decision-manager';
+import AccessDecisionManager  from '../../../access-decision-manager/src';
 import AccessDecisionManagerProvider from '../access-decision-manager-provider';
 
 describe('src', () => {
diff --git a/packages/access-decision-manager-express/src/access-decision-manager-provider.ts b/packages/access-decision-manager-express/src/access-decision-manager-provider.ts
index 9694602..9fa806b 100644
--- a/packages/access-decision-manager-express/src/access-decision-manager-provider.ts
+++ b/packages/access-decision-manager-express/src/access-decision-manager-provider.ts
@@ -1,6 +1,6 @@
 // eslint-disable-next-line import/no-extraneous-dependencies
 import { NextFunction, Request, Response } from 'express';
-import AccessDecisionManager, { Voter } from '@wizeline/access-decision-manager';
+import AccessDecisionManager, { Voter } from '../../../packages/access-decision-manager/src';
 
 // eslint-disable-next-line @typescript-eslint/no-explicit-any
 type ADMArgumentFactory = (req: Request) => any;
`

but that it's not what we want to do, I will check the real solution but this was working in our latest CI/CD -> https://app.travis-ci.com/ (check this out)